exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/StorageResources.java

```diff
@@ -137,31 +159,37 @@ public JsonResult enablePlugin(@PathParam("name") String name, @PathParam("val")
   }
 
   @GET
-  @Path("/storage/{name}/export")
+  @Path("/storage/{name}/export/{format}")
   @Produces(MediaType.APPLICATION_JSON)
-  public Response exportPlugin(@PathParam("name") String name) {
-    Response.ResponseBuilder response = Response.ok(getStoragePluginJSON(name));
-    response.header("Content-Disposition", String.format("attachment;filename=\"%s.json\"", name));
-    return response.build();
+  public Response exportPlugin(@PathParam("name") String name, @PathParam("format") String format) {
+    if (JSON_FILE_NAME.equalsIgnoreCase(format) || HOCON_FILE_NAME.equalsIgnoreCase(format)) {
+      PluginConfigWrapper storagePluginConfigs = getPluginConfig(name);
+      Response.ResponseBuilder response = Response.ok(storagePluginConfigs);
+      response.header("Content-Disposition", String.format("attachment;filename=\"%s.%s\"", name, format));
```

Comment:
Please use ```javax.ws.rs.core.HttpHeaders``` for header name constants here and in other places.

Reply:
I applied it also in `LogsResources.java` class